packages/react-native-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/react-native-sdk",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

Comment:
We were creating alpha versions before deploying in `prod`. NABD, just for alignment.